wasm/jit/jit_amd64_test.go

```diff
@@ -1404,6 +1402,58 @@ func TestAmd64Builder_handleCall(t *testing.T) {
 	})
 }
 
+func TestAmd64Builder_handleMemoryGrow(t *testing.T) {
+	builder := requireNewBuilder(t)
+
+	builder.initializeReservedRegisters()
+	// Emit memory.grow instructions.
+	builder.handleMemoryGrow()
+
+	// Compile.
```

Comment:
I see this often.. a comment compile for a function assemble.. should we rename assemble to compile?

Reply:
https://github.com/tetratelabs/wazero/pull/85/commits/4449034a82cb32489f3aec84601ae80b6029f9ae agreed so renamed! :D